### ISSUE TITLE
Event Limit? (Kodi 18)

### ISSUE THREAD

**Issue by ozzymanborn** [sterd71/plugin.video.eurosporton#16]
In Olympic coverage list there is some events afternoon and they're not appearing. As there is too many events it might be page 2 thing.

I hope if you're update this still would be a kodi 18 version also.

**Comment by pivoj13** [comment]
I have the same issue - seems that current limit is 150 events. Is it possible to change this somewhere?

**Comment by Jordan23M** [comment]
Yes only shows max 100 event limit per day on Kodi 19, also this not allow to see event "ON TV" if not showed in the list. Would be great if someone could fix or tell us how to modify this in the source . thanks

**Comment by shamanNS** [comment]
Proper paginating of requests needs to be added, there are no hardcoded item limits in this addon code but after a quick check it looks like server API has page size limit of 200 items max per one request. And you get even less info if you don't send "&page[items.size]=200"

@sterd71

**Comment by sterd71** [comment]
All, (@Jordan23M, @ozzymanborn, @shamanNS, @pivoj13)

I've released a temporary fix for Kodi 18 & Kodi 19.

This doesn't include paging yet, but I'm working on that now.

Eurosport are using completely different URL's for their Olympics coverage, which is why no items were showing under 'On TV'.

I've added two new menu items for the Olympics: On now, and On today.  On today will still be limited until I can get the paging working.

When the games are over I will remove these two menu items.
